epam_tr/src/main/java/com/epam/tr/entities/FileType.java

```diff
@@ -0,0 +1,5 @@
+package com.epam.tr.entities;
+
+public enum FileType {
+    FILE,FOLDER,DRIVE
```

Comment:
add spaces

Reply:
FileType may be Folder - but folder is not a file you know. So enum name does not seem to be appropriate.

Reply:
I would move the enum inside FileEntity and name Type.